### DISCUSSION TITLE
Help with markdown

### DISCUSSION THREAD

**Discussion by JaedenArmstrong** [community/community#130941] [New to GitHub]
Relatively new to GitHub, and I need some quick help please...

**1.** How can remove the line space below an H6 heading, if it's possible?
   I tried using **{lines-below=0}** but it didn't work, instead the entire **{lines-below=0}** appeared in the committed document on my fork.

**2.** How can I remove the line space above a numbered list, if it's possible?
**3.** How do I indent an entire INCLUDE? Is it done in the document? How? Or is it done in the INCLUDE? How?

**Comment by mfm-builder-001** [comment]
Regarding your attempt with {lines-below=0}, that syntax is not a standard Markdown feature. It's possible that it's a custom extension or a feature specific to the platform or tool you're using. In general, Markdown doesn't have a built-in way to directly control the spacing above or below headings or lists.

**Reply by JaedenArmstrong** [reply]
Thanks very much for your kind a friendly reply, it's much appreciated and rare that people respond without flaming the question, so my respect to you.

So, just to be certain, there is no workaround, or solution? I am trying to create a very specific document formatting according to clearly defined needs for a particular MS Learn repository.
